Source/WTF/wtf/WeakHashMap.h

```diff
@@ -369,9 +369,11 @@ class WeakHashMap final {
     static WeakPtrImpl<Counter>* keyImplIfExists(const T& key)
     {
         auto& weakPtrImpl = key.weakPtrFactory().m_impl;
-        if (!weakPtrImpl || !*weakPtrImpl)
-            return nullptr;
-        return weakPtrImpl.get();
+        if (auto* pointer = weakPtrImpl.pointer()) {
```

Comment:
Why not: auto* pointer = weakPtrImpl.pointer(); pointer && *pointer?

Reply:
That sounds good. Changed.